src/System.Runtime/src/System/Reflection/AssemblyNameProxy.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+**
+**
+**
+**
+**
+** Purpose: Remotable version the AssemblyName
+**
+**
+===========================================================*/
+namespace System.Reflection {
+    using System;
+    using System.Runtime.Versioning;
+
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public class AssemblyNameProxy : MarshalByRefObject
```

Comment:
It is, but AssemblyName.GetAssemblyName(string) method was not exposed either on CoreCLR, as it called a runtime method without implementation.